examples/xgboost/xgboost_sklearn/train_sklearn.py

```diff
@@ -0,0 +1,44 @@
+from pprint import pprint
```

Comment:
Nit: can we rename this file to `train.py` because other examples use this convetion?